app/scripts/modules/core/delivery/executionGroup/execution/execution.component.html

```diff
@@ -0,0 +1,115 @@
+<div class="execution" id="execution-{{::$ctrl.execution.id}}" ng-class="$ctrl.showingDetails ? 'show-details' : 'details-hidden'">
+  <div class="execution-overview group-by-{{::$ctrl.sortFilter.groupBy}}">
+    <h4 class="execution-name" ng-if="::$ctrl.sortFilter.groupBy !== 'name'">
+      <account-label-color ng-repeat="account in $ctrl.execution.deploymentTargets" account="{{account}}"></account-label-color>
+      {{::$ctrl.execution.name}}
+    </h4>
+    <execution-status execution="::$ctrl.execution"
+                      toggle-details="$ctrl.toggleDetails(node)"
```

Comment:
Angular components pass properties via a map when bound with an `&`. Like https://docs.angularjs.org/guide/component. It's confusing.